ci-conda.Dockerfile

```diff
@@ -87,9 +109,9 @@ case "${LINUX_VER}" in
         tzdata_pkgs=(tzdata)
     fi
 
-    apt-get update
-    apt-get upgrade -y
-    apt-get install -y --no-install-recommends \
+    rapids-retry apt-get update
+    rapids-retry apt-get upgrade -y
+    rapids-retry apt-get install -y --no-install-recommends \
```

Comment:
I added `rapids-retry` to all `apt-get` commands that happen _before_ we set up the apt retry policy. For `apt-get` commands after the apt retry policy is set, I didn't bother.

Should we instead move the apt retry policy to the top of the Dockerfile, so that we never have to prefix `rapids-retry apt-get` and rely on apt's own retry system?

Reply:
And if we set the `apt` retry settings first, we'll also reduce the chance of that initial install of `wget` failing

Reply:
case in point: https://github.com/rapidsai/ci-imgs/actions/runs/15734452777/job/44344936107?pr=288